api/v2/api.go

```diff
@@ -53,6 +54,7 @@ type API struct {
 	peer           *cluster.Peer
 	silences       *silence.Silences
 	alerts         provider.Alerts
+	groups         groupsFn
```

Comment:
For me this is ambiguous, does it return groups of alerts, silences, receivers, apples, ...? How about `alertGroups`?

Reply:
Apples would be nice. Will update :)